docs/.vitepress/theme/style.css

```
@@ -0,0 +1,473 @@
+ * Accent:       #7ebab5 (teal)
+ * Primary text: #f6f5f5 (warm white) / #1a1c1e (dark ink)
+ * Dark BG:      #090a0c (deep midnight)
+ * Light BG:     #f8f9fa (warm paper)
+ * Secondary:    #353a40 (slate)
+ *
+ * This file is designed to be reusable across all Phenotype
+ * VitePress projects. Copy or symlink to your project's
+ * .vitepress/theme/custom.css
+ * ═══════════════════════════════════════════════════════════════ */
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap');
```

Stylelint (17.12.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap'" (import-notation)

(import-notation)